.github/CONTRIBUTING.md

```diff
@@ -26,6 +26,11 @@ With the exception of very small typos, all changes to this repository generally
 If this is your first time contributing, consider searching for [unassigned issues that also have the `good first issue` label](https://github.com/OpenContributionsProject/opencontributionssite/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22+label%3A%22good+first+issue%22+no%3Aassignee).
 If the issue you'd like to fix isn't found on the issue, see [Reporting Issues](#reporting-issues) for filing your own (please do!).
 
+### Assigning an issue
+
+Once you found an unassigned issue that you'd like to work on, you can self-assign the issue by commenting ".take" on the issue. This will trigger an automate GitHub Action to run and assign you to the issue. If successful, you will receive an automated comment that says, "Thanks for taking this issue! Let us know if you have any questions!" Subsequently, you can get started on the task! See the image below for an example of what this should look like: 
```

Comment:
Just nitpicking: a few small proofreading suggestions to trim down a teeny bit, if you like:

```suggestion
Once you've found an unassigned issue that you'd like to work on, you can self-assign the issue by commenting _".take"_ on the issue. This will trigger an automated GitHub Action t assign you to the issue and post an automated comment that says, _"Thanks for taking this issue! Let us know if you have any questions!"_ Subsequently, you can get started on the task! See the image below for an example of what this should look like: 
```

I think it'd be totally fine to ignore me on those nitpicks. 😄 

Separately, I'm surprised the `lint:md` task isn't failing here 🤔. It fails locally for me with:

```plaintext
josh ~/repos/opencontributionssite $ pnpm lint:md

> site@0.1.0 lint:md /Users/josh/repos/opencontributionssite
> markdownlint "**/*.md" ".github/**/*.md" --rules sentences-per-line

.github/CONTRIBUTING.md:30 sentences-per-line Each sentence should be on its own line [Context: "e issue. If su"]
.github/CONTRIBUTING.md:30 sentences-per-line Each sentence should be on its own line [Context: "e issue. This "]
.github/CONTRIBUTING.md:31:457 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
 ELIFECYCLE  Command failed with exit code 1.
```

If the PR is passing then 🤷 we can merge and I can file a followup issue to investigate.